src/Charcoal/Property/AbstractProperty.php

```diff
@@ -302,11 +303,26 @@ public function l10nIdent($lang = null)
         return $this;
     }
 
+    /**
+     * Clear the property's value.
+     *
+     * @deprecated
+     *
+     * @return self
+     */
+    final public function clearVal()
```

Comment:
Where is it needed?

Reply:
https://github.com/locomotivemtl/charcoal-core/pull/7/files#diff-cca8f65696df7f6f5a4268eed48b85c5R31